dubbo-common/src/main/java/org/apache/dubbo/common/compiler/support/ClassUtils.java

```diff
@@ -431,5 +431,13 @@ public static String getInitCode(Class<?> type) {
         }
         return map;
     }
+    
+    /**
+     * get classname from qualified name
+     */
+    public static String getSimpleClassName(String qualifiedName) {
+        int i = qualifiedName.lastIndexOf('.');

```

Comment:
Better to check and have check for non empty or null string 

`org.apache.dubbo.common.utils.Assert.notEmptyString(qualifiedName, "Name can't be null or blank");`

Reply:
thank you for your advice

Reply:
committed